package.json

```diff
@@ -32,47 +31,39 @@
     "takeshape:post-checkout-hook": "shape post-checkout-hook",
     "takeshape:post-merge-hook": "shape post-merge-hook",
     "takeshape:promote-branch": "shape promote-branch",
-    "test": "jest",
-    "test:cov": "jest --coverage",
-    "test:watch": "jest --watch",
+    "test": "vitest run",
+    "test:watch": "vitest",
     "typecheck": "tsc --noEmit --pretty"
   },
   "dependencies": {
-    "@apollo/client": "^3.9.6",
+    "@apollo/client": "^3.9.7",
+    "@apollo/experimental-nextjs-app-support": "^0.8.0",
+    "@auth/core": "^0.28.0",
     "@headlessui/react": "^1.7.18",
     "@heroicons/react": "^2.1.1",
-    "@next/bundle-analyzer": "^13.5.6",
-    "@next/env": "^13.5.6",
-    "@sentry/nextjs": "^7.106.1",
-    "@sentry/node": "^7.106.1",
-    "@sentry/react": "^7.106.1",
-    "@takeshape/next-auth-all-access": "^1.0.0",
+    "@next/bundle-analyzer": "^14.1.3",
+    "@next/env": "^14.1.3",
+    "@sentry/nextjs": "^7.107.0",
+    "@takeshape/next-auth-all-access": "^1.1.0",
     "@takeshape/routing": "^10.7.6",
-    "abortcontroller-polyfill": "^1.7.5",
-    "card-validator": "^9.1.0",
     "cookies-next": "^4.1.1",
-    "date-fns": "^3.4.0",
+    "cssnano": "^6.1.0",
+    "date-fns": "^3.6.0",
     "graphql": "^16.8.1",
     "imask": "^7.5.0",
-    "jotai": "^2.7.0",
-    "next": "^13.5.6",
-    "next-auth": "^4.24.7",
-    "next-plugin-preval": "^1.2.5",
-    "next-pwa": "5.6.0",
+    "jotai": "^2.7.1",
+    "next": "^14.1.3",
+    "next-auth": "^5.0.0-beta.15",
```

Comment:
This version truly supports the `app` folder, and since I'm spending time on this now it made more sense to move to it even though it is in beta. 

We should update this when there is a true release.